File: Product of Digits/product_of_digits.c
#include <stdio.h>
#include <stdlib.h>
int main() {
    int n;
    printf("Enter a number: ");
    scanf("%d", &n);
    int temp = abs(n);
    int product = 1;
    while (temp > 0) {
        product = product * (temp % 10);
        temp = temp / 10;
    }
    printf("Product of digits: %d\n", product);
    return 0;
}

File: Product of Digits/ProductOfDigits.java
public class ProductOfDigits {
    public static void main(String[] args) {
        java.util.Scanner sc = new java.util.Scanner(System.in);
        System.out.print("Enter a number: ");
        int n = sc.nextInt();
        int temp = Math.abs(n);
        int product = 1;
        while (temp > 0) {
            product = product * (temp % 10);
            temp = temp / 10;
        }
        System.out.println("Product of digits: " + product);
    }
}
